examples/kube_prometheus_stack/values-prometheus.yaml

```diff
@@ -497,7 +499,8 @@ prometheus-node-exporter:
   podLabels: {}
   ## Additional annotations for pods in the DaemonSet
   podAnnotations: {}
-  resources: {}
+  resources:
+    {}
```

Comment:
why is it formatted differently?
because of comments below?

Reply:
Yeah, I think so.